collectd.conf.tmpl

```diff
@@ -1,5 +1,8 @@
 Interval 60
 
+# The Stackdriver Agent does not rely on FQDNLookup to determine the hostname.
```

Comment:
`The Stackdriver agent does not use fully qualified domain names.`

Reply:
PTAL